site/content/docs/contributing/1.0-roadmap.md

```diff
@@ -69,12 +69,12 @@ Longer Term goals include:
 - Enabling a suitable local storage provider for testing applications that need
 persistent storage
 
-## Misc:
+## Misc
```

Comment:
this is possibly the only change that actually seems helpful
blocking on extra spaces that do not affect the rendering seems pretty pointless, and is most the change here :/

I don't like the idea of contributors fighting with the linter over pointless details like that :(

Reply:
we get a lot of really helpful drive-by contributions here from totally new contributors to the docs in particular to clarify things they've tripped up on, it would be a shame to trip them up further.